### ISSUE TITLE
Drop support SQLAlchemy 0.4

### ISSUE THREAD

**Issue by anton44eg** [fixture-py/fixture#21]
I'm fixing tests. There some troubles with SQLAlchemy 0.4 and SQLite, it closes connection to DB.

SQLAlchemy 0.4 is very old so my proposal is drop support of it.

**Comment by Trii** [comment]
This project never gets updates anymore. It seems the owner is gone or something.

**Comment by anton44eg** [comment]
Yeah, only 2 commits last year. Maybe you can advice other fixture project for SQLAlchemy?

**Comment by Trii** [comment]
I don't know of any that have as good of an API. I use this with Google App
Engine but I have to make my own fixes

**Comment by kumar303** [comment]
I am the original owner but I don't use the module anymore. A couple
different people were maintaining it for a while but I'm not sure what
their status is. If you have a pull request that solves it I can get it
merged in for you. I think deprecating the ancient sqlalchemy version would
be fine.
